example/ci/pipeline.yaml

```diff
@@ -0,0 +1,118 @@
+resources:
+  - name: source
+    type: git
+    source:
+      uri: ((git_source))
+      private_key: ((privatekey))
+      branch: ((git_branch))
+      skip_ssl_verification: ((git_skip_ssl_verification))
+      # This enables triggering on tag changes that match the glob pattern
+      # tag_filter: "*.*.*"
+
+  - name: image-bb-gitversion-tool
+    type: docker-image
+    source:
+      repository: elbb/bb-gitversion
+      tag: ((bb_gitversion_version))
+
+  - name: s3-gitversion
+    type: s3
+    source:
+      bucket: ((bb_name))
+      endpoint: ((s3_endpoint))
+      # at least one regex capture group is needed here
+      regexp: version/gitversion-(.*).tar.gz
```

Comment:
i get s3-gitversion check errors, because
```regexp: version/gitversion-(.*).tar.gz``` doesnt match ```file: gen/gitversion-*.tar.gz ```

(I solved that in my bb-gitversion MR, where i names the director `gitversion` and adapted `regexp` + `file`.)

Reply:
good catch! Thanks. fixed.